### PR TITLE
Use new python openshift version 0.13.1

### PR DESCRIPTION
**Pull request by mpryc**
Fixes Issue #262, which uses newer version of OpenShift python module.

related to #446 

@redhat-cop/mdt

### CODE REVIEW

**Review by weshayutin — APPROVED** [review]
VERY NICE! WFM :)

File: exporters/pelorus/utils.py [hunk 1]
```diff
@@ -198,3 +201,25 @@ def get_env_var(var_name: str, default_value: Optional[str] = None) -> Optional[
...
+        k8s_client = DynamicClient(k8sconfig)
+    except config.config_exception.ConfigException:
+        # Try load config from cluster
+        config.load_incluster_config()
+        file_namespace = open(
+            "/run/secrets/kubernetes.io/serviceaccount/namespace", "r"
+        )
+        if file_namespace.mode == "r":
+            namespace = file_namespace.read()
+            print("namespace: %s\n" % (namespace))
```
Comment (KevinMGranger) [inline exporters/pelorus/utils.py:219]: I'm wondering why this was here in the first place-- just printing what namespace the service account is in? Can't that just be checked in OpenShift directly?

If we keep this, then it looks like the old code had some issues:

1. It's checking that the file's mode is `r`, when it just opened it with `r`.
2. The file never gets closed.

We can just replace this whole part with `namespace = Path("/run/secrets/kubernetes.io/serviceaccount/namespace").read_text()`

File: exporters/requirements.txt [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # Used by all exporters
-openshift < 0.12       # module openshift
+openshift == 0.13.1    # module openshift
```
Comment (KevinMGranger) [inline exporters/requirements.txt:2]: 🎉

File: exporters/committime/app.py [hunk 3]
```diff
@@ -45,10 +43,7 @@ def getCollector(
 if __name__ == "__main__":
     pelorus.upgrade_legacy_vars()
 
-    pelorus.load_kube_config()
-    k8s_config = client.Configuration()
-    k8s_client = client.api_client.ApiClient(configuration=k8s_config)
-    dyn_client = DynamicClient(k8s_client)
+    dyn_client = pelorus.utils.get_k8s_client()
```
Comment (KevinMGranger) [inline exporters/committime/app.py:46]: I like this, it really cleans things up!

**Comment by mpryc** [comment]
Merging, got approval from @weshayutin comment from @KevinMGranger was resolved and 4.9 prow is a known issue not related to this change.

**Pull request merged**